misk-tailwind/src/main/kotlin/misk/tailwind/TailwindHtmlLayout.kt

```diff
@@ -26,12 +26,14 @@ fun TagConsumer<*>.TailwindHtmlLayout(appRoot: String, title: String, playCdn: B
         type = "image/x-icon"
         href = "/static/favicon.ico"
       }
-      if (playCdn) {
+      // TODO add Gradle plugin to comb through service JAR to build minified Tailwind CSS
+      // Until then, use play CDN so all CSS is present for UI from Misk or internal libaries/services
+//      if (playCdn) {
```

Comment:
Nit: it's kind of deceptive that the `playCdn` parameter now does nothing, but I recognize that it'd be a pain to delete it and then re-add it just for an intermediary state, so 👍

Reply:
Agreed, thanks for understanding. Aiming for it to only be a temporary change.